runtime/opt/zalando/init.d/10-prepare-disks.py

```diff
@@ -113,13 +113,15 @@ def format_partition(partition, filesystem="ext4", initialize=False, is_already_
         LOG.info("Nothing to do for disk %s", partition)
 
 
-def mount_partition(partition, mountpoint, dir_exists=None, is_mounted=None):
+def mount_partition(partition, mountpoint, options, dir_exists=None, is_mounted=None):
     """Mounts formatted disks provided by /etc/taupage.yaml"""
-    if is_mounted is False and dir_exists is False:
-        os.makedirs(mountpoint)
-        subprocess.check_call(["mount", partition, mountpoint])
-    elif is_mounted is False and dir_exists is True:
-        subprocess.check_call(["mount", partition, mountpoint])
+    if is_mounted is False:
+        if dir_exists is False:
+            os.makedirs(mountpoint)
+        if options and len(options) > 0:
+            subprocess.check_call(["mount", "-o", options.replace(' ',''), partition, mountpoint])
```

Comment:
Makes sense, changed the code to prepare the list of arguments and call mount only once.